src/components/item/form/BaseItemForm.tsx

```diff
@@ -13,9 +13,15 @@ type Props = {
   updatedProperties: Partial<DiscriminatedItem<UnknownExtra>>;
   onChange: (props: Partial<DiscriminatedItem>) => void;
   item: Partial<ItemRecord>;
+  required?: boolean;
 };
 
-const BaseForm: FC<Props> = ({ onChange, item, updatedProperties }) => {
+const BaseForm: FC<Props> = ({
+  onChange,
+  item,
+  required,
```

Comment:
I don't think this is a good name. Maybe you should call this prop `isNameRequired`?